tests/datatypes/test_bytes.py

```diff
@@ -231,6 +231,16 @@ def test_contains(self):
             print(['b', 'e'] in b'pybee') #Test TypeError invalid byte array
         """, exits_early=True)
 
+    def test_center(self):
+        self.assertCodeExecution("""
+            print(b'pybee'.center(5))
+            print(b'pybee'.center(True, b'a'))
+            print(b'pybee'.center(13, b'a'))
+            """)
+        self.assertCodeExecution("""
+            print([].center) #Test TypeError invalid byte array
```

Comment:
This is an attribute operation on an array - it has nothing to do with byte arrays, and it isn't invoking a method.